extensions/src/main/java/org/pentaho/platform/web/http/api/resources/RolesWrapper.java

```diff
@@ -0,0 +1,48 @@
+/*!
```

Comment:
@mbatchelor I did it because we cant change the format of existing xml response.  Please look at https://help.pentaho.com/Documentation/7.0/0R0/070/010/0D0/0N0#path__userrolelist_getRolesForUser.html. The method should return`<roles>
<role>Role1</role>
<role>Role2</role>
</roles>`. But if we use RoleListWrapper it will return `<roleList>
<roles>Role1</roles>
<roles>Role2</roles>
</roleList>` . You may look at example https://help.pentaho.com/Documentation/7.0/0R0/070/010/0D0/0N0#path__userrolelist_extraRoles.html . These two rest endpoints ( getRolesForUser and getUsersInRole) have a different format of xml from  other endpoints in this class.